tcptracer-bpf.c

```diff
@@ -821,6 +841,48 @@ int kretprobe__inet_csk_accept(struct pt_regs *ctx)
 	return 0;
 }
 
+SEC("kprobe/fd_install")
+int kprobe__fd_install(struct pt_regs *ctx)
+{
+	u64 pid = bpf_get_current_pid_tgid();
+	u32 tgid = pid >> 32;
+	unsigned long fd = (unsigned long) PT_REGS_PARM1(ctx);
```

Comment:
I know we are not using bcc but I was assuming we were using the same C compiler. We don't necessarily need to change that, but I would like to understand the differences. Mind elaborating?